release notes/v0.42.0.md

```diff
@@ -0,0 +1,151 @@
+k6 v0.42.0 is here and it brings a few exciting updates! :tada:
+
+## New Features
+
+### Experimental Prometheus Remote Write Output [#2784](https://github.com/grafana/k6/pull/2784)
+
+Added a new builtin Output to any [Prometheus Remote Write](https://docs.google.com/document/d/1LPhVRSFkGNSuU1fBd81ulhsCPR4hkSZyyBj1SZ8fWOM/edit) implementations (e.g. [Prometheus](https://prometheus.io/docs/prometheus/latest/feature_flags/#remote-write-receiver), [Mimir](https://grafana.com/docs/mimir/latest/operators-guide/reference-http-api/#remote-write)). This is an experimental API and it could change in the future introducing breaking changes.
```

Comment:
```suggestion
Added a new builtin Output to any [Prometheus Remote Write](https://docs.google.com/document/d/1LPhVRSFkGNSuU1fBd81ulhsCPR4hkSZyyBj1SZ8fWOM/edit) implementations (e.g. [Prometheus](https://prometheus.io/docs/prometheus/latest/feature_flags/#remote-write-receiver), [Mimir](https://grafana.com/docs/mimir/latest/operators-guide/reference-http-api/#remote-write)). This is an experimental feature and it could change in the future introducing breaking changes.
```

The k6 parts is not an API arguably ;)

Reply:
For API I meant the k6 CLI's commands..for example removing the `experimental-` or adding the feature for setting the string URI. I will try to rephrase.